build-farm/platform-specific-configurations/aix.sh

```diff
@@ -68,7 +68,22 @@ then
     fi
   fi
   export JDK_BOOT_DIR=$JDK10_BOOT_DIR
+fi
 
+if [ "${JAVA_TO_BUILD}" == "${JDK12_VERSION}" ] || [ "${JAVA_TO_BUILD}" == "${JDKHEAD_VERSION}" ];
+then
+  if [ ! -d "$JDK11_BOOT_DIR" ]; then
+    export JDK11_BOOT_DIR="$PWD/jdk-11"
+    if [ ! -d "$JDK11_BOOT_DIR/bin" ]; then
+      mkdir -p "$JDK11_BOOT_DIR"
+      wget -q -O - "https://api.adoptopenjdk.net/v2/binary/releases/openjdk11?os=aix&release=latest&arch=${ARCHITECTURE}&heap_size=normal&openjdk_impl=openj9" | tar xpzf - --strip-components=2 -C "$JDK11_BOOT_DIR"
+    fi
+  fi
+  export JDK_BOOT_DIR=$JDK11_BOOT_DIR
+fi
+
+if [ "${JAVA_TO_BUILD}" == "${JDK11_VERSION}" ] || [ "${JAVA_TO_BUILD}" == "${JDK12_VERSION}" ] || [ "${JAVA_TO_BUILD}" == "${JDKHEAD_VERSION}" ];
```

Comment:
@lumpfish So we don't set these for building Java 8?  What does Java 8 default to then?

Reply:
Not something I touched I hope, but at the top of the files are settings like ``export PATH="/opt/freeware/bin:/usr/local/bin:/opt/IBM/xlC/13.1.3/bin:/opt/IBM/xlc/13.1.3/bin:$PATH"
export CONFIGURE_ARGS_FOR_ANY_PLATFORM="${CONFIGURE_ARGS_FOR_ANY_PLATFORM} --with-memory-size=18000 --with-cups-include=/opt/freeware/include"```.  I assume that since there are no specific gcc settings for jdk8 and hotspot the build uses the default compiler on the build machine?

Reply:
If we could setup and test openjdk8-pipeline running from by branch (it's a bit awkward, but can be done - I couldn't manage to get it working under the 'work in progress' tab though) we could build the other releases and make sure they're OK.

Reply:
Ah yes the default should work in that case - we'll refactor to make everything explicit in a later PR